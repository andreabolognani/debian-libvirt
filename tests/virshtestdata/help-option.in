help echo
echo --help
echo --help=data
echo --help data
