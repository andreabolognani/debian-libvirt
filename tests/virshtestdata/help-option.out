  NAME
    echo - echo arguments. Used for internal testing.

  SYNOPSIS
    echo [--shell] [--xml] [--split] [--err] [--prefix <string>] [<string>]...

  DESCRIPTION
    Echo back arguments, possibly with quoting. Used for internal testing.

  OPTIONS
    --shell          escape for shell use
    --xml            escape for XML use
    --split          split each argument on ','; ',,' is an escape sequence
    --err            output to stderr
    --prefix <string>  prefix the message
    [--string] <string>...  arguments to echo
  NAME
    echo - echo arguments. Used for internal testing.

  SYNOPSIS
    echo [--shell] [--xml] [--split] [--err] [--prefix <string>] [<string>]...

  DESCRIPTION
    Echo back arguments, possibly with quoting. Used for internal testing.

  OPTIONS
    --shell          escape for shell use
    --xml            escape for XML use
    --split          split each argument on ','; ',,' is an escape sequence
    --err            output to stderr
    --prefix <string>  prefix the message
    [--string] <string>...  arguments to echo
  NAME
    echo - echo arguments. Used for internal testing.

  SYNOPSIS
    echo [--shell] [--xml] [--split] [--err] [--prefix <string>] [<string>]...

  DESCRIPTION
    Echo back arguments, possibly with quoting. Used for internal testing.

  OPTIONS
    --shell          escape for shell use
    --xml            escape for XML use
    --split          split each argument on ','; ',,' is an escape sequence
    --err            output to stderr
    --prefix <string>  prefix the message
    [--string] <string>...  arguments to echo
  NAME
    echo - echo arguments. Used for internal testing.

  SYNOPSIS
    echo [--shell] [--xml] [--split] [--err] [--prefix <string>] [<string>]...

  DESCRIPTION
    Echo back arguments, possibly with quoting. Used for internal testing.

  OPTIONS
    --shell          escape for shell use
    --xml            escape for XML use
    --split          split each argument on ','; ',,' is an escape sequence
    --err            output to stderr
    --prefix <string>  prefix the message
    [--string] <string>...  arguments to echo
